README.Rmd

```diff
@@ -26,75 +26,91 @@ knitr::opts_chunk$set(
 [![Coverage Status](https://img.shields.io/codecov/c/github/r-lib/pak/master.svg)](https://codecov.io/github/r-lib/pak?branch=master)
 <!-- badges: end -->
 
-pak installs R packages from various sources.
+pak installs R packages from various sources. It is a replacement for
+`install.packages()` and `devtools::install_github()`.
+pak is fast, safe and convenient.
+
+<p align="center">
+	<img width="1000" src="https://cdn.jsdelivr.net/gh/r-lib/pak@readme/tools/images/install-2.svg">
+</p>
+
 
 ## Installation
 
 Install the package from CRAN:
 
-``` r
+```r
 install.packages("pak")
 pak::pak_create_private_lib()
 ```
 
 The second line creates pak's own package library, to avoid interference
-between pak's dependencies and the user's regular packages. Run this
-command any time to update pak's package library.
+between pak's dependencies and the user's regular packages.
```

Comment:
So, if someone fails to do this -- run `pak::pak_create_private_lib()` after install -- will pak let them know it's needed upon first use?

I feel like you could use even clearer wording somewhere, maybe here, that this design gives pak its own ... "oxygen supply", like a pilot on a plane or something. Is there a better metaphor? This eliminates a whole category of package installation headaches. I would make an explicit mention of the known gotchas with corrupt installs on Windows. Right now the connection is pretty subtle and some people might miss it.

Reply:
> will pak let them know it's needed upon first use?

Then pak runs it automatically.

I agree we could elaborate this, but maybe the README is not the best place for it? I'll also start writing a vignette, maybe there?

Reply:
If it's run automatically, do we even need to mention it here then?

That is, instead of mentioning here and elaborating in vignette, maybe do it the other way around.

Reply:
> If it's run automatically, do we even need to mention it here then?

It is run automatically, but it is good to run it straight after installation, because then pak can just copy its dependencies from the regular library, they must be available locally. Otherwise if they are not available locallt, pak will try to download them, and that can go wrong.

It is likely that we'll need to fine-tune how the private library is created, so I don't really want to go into the details here.  If they run it after installation, that's good. If they don't that's probably OK as well.

I think i'll rename this to `pak_setup()`, that is less scary, and remove it  from here.

Reply:
Yeah, `pak_setup()` would be much better. And I think you could mention it parenthetically, i.e. "(You might also want to run `pak::pak_setup()`; it'll be run automatically when needed but you might want to do it now to save some time later)"